scripts/drone/steps/lib.star

```diff
@@ -775,11 +775,32 @@ def lint_frontend_step():
         "commands": [
             "yarn run prettier:check",
             "yarn run lint",
-            "yarn run i18n:compile",  # TODO: right place for this?
             "yarn run typecheck",
         ],
     }
 
+def verify_i18n_step():
+    return {
+        "name": "verify-i18n",
+        "image": build_image,
+        "depends_on": [
+            "yarn-install",
+        ],
+        "commands": [
+            "yarn run i18n:extract",
+            # Verify that translation extraction has been committed
+            '''
+            file_diff=$(git diff -- locales --stat)
+            if [ -n "$file_diff" ]; then
+                echo $file_diff
+                echo "Translation extraction has not been committed. Please commit the changes and push again."
+                exit 1
+            fi
```

Comment:
This whole part is visible in the the output regardless if it succeeds or fails. Should it be extracted to a script file?